core_server/tasks_settings.py

```diff
@@ -23,4 +23,8 @@
             hour="*/2",  # do it every 2 hours  # TODO: maybe have this from env variable
         ),
     },
+    "calculate_model_metrics": {
+        "task": "core.tasks.model_monitoring.calculate_model_metrics",
+        "schedule": crontab(hour="2", minute="46"),
```

Comment:
Any specific reasons to use hour 2 and minute 46?

Reply:
@bewakes this will be chaged to 0 hour 0 min when everything is ready. It is added just for testing for now.

Reply:
You can mark PR as draft in that case.